storybook/pages/AboutViewPage.qml

```diff
@@ -44,7 +44,7 @@ SplitView {
                 }
 
                 function qtRuntimeVersion() {
-                    return SystemUtils.qtRuntimeVersion()
+                    return SQCore.SystemUtils.qtRuntimeVersion()
```

Comment:
`SystemUtils` is part of both `StatusQ.Core 0.1` and `Storybook` imports and this unscoped access stopped working in Qt 6.9 :(